core/object/object.h

```diff
@@ -1052,6 +1052,22 @@ class ObjectDB {
 
 		return object;
 	}
+
+public:
+	typedef void (*DebugFunc)(Object *p_obj);
+
+	_ALWAYS_INLINE_ static Object *get_instance(ObjectID p_instance_id) {
+		return _get_instance(p_instance_id);
+	}
+
+	template <typename T>
+	_ALWAYS_INLINE_ static T *get_instance(ObjectID p_instance_id) {
+		return Object::cast_to<T>(_get_instance(p_instance_id));
+	}
+
+	template <typename T>
+	_ALWAYS_INLINE_ static Ref<T> get_ref(ObjectID p_instance_id); // Defined in ref_counted.h
```

Comment:
This is pretty unusual (defining a function in another object file than the header itself), so I'm very grateful for your comment.

To me this screams defining `ObjectDB` outside `object.h` (into its own header), but this is definitely out of scope for this PR.
So I think your workaround is acceptable. I certainly can't think of a better solution 😅